src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_client_factory.py

```diff
@@ -3,42 +3,52 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+from azure.cli.core.commands.client_factory import configure_common_settings
+from azure.cli.core.commands.client_factory import get_mgmt_service_client
 
-def _auth_client_factory(scope=None):
+
+def cf_compute_service(cli_ctx, _):
```

Comment:
Change `_` to `*_` then you don't need to call it like `cf_compute_service(cli_ctx, None)` as you can just do `cf_compute_service(cli_ctx)`. Same with the others.